VERSION

```diff
@@ -1 +1 @@
-v0.5.0
+v0.7.0
```

Comment:
I think this file can just be dropped (along with Makefile uses of it), it's just a library now.